main.go

```diff
@@ -116,3 +129,50 @@ Arguments:
 		helpTab,
 	)
 }
+
+// This section translates a windows path such as "C:\Users\user" to "/Users/user"
+// and removes the default paths for mingw and git4windows to issues when you
+// specify a file path to start code-server in.
+func gitbashWindowsDir(dir string) string {
+	if dir == "~" { //Special case
+		return "~/"
+	}
+	mountPoints := gitbashMountPointsAndHome()
+
+	// Apply mount points
+	absDir, _ := filepath.Abs(dir)
+	absDir = filepath.ToSlash(absDir)
+	for _, mp := range mountPoints {
+		if strings.HasPrefix(absDir, mp[0]) {
+			resolved := strings.Replace(absDir, mp[0], mp[1], 1)
+			flog.Info("Resolved windows path '%s' to '%s", dir, resolved)
+			return resolved
+		}
+	}
+	return dir
+}
+
+// This function returns an array with MINGW64 mount points including relative home dir
+func gitbashMountPointsAndHome() [][]string {
+	// Initialize mount points with home dir
+	mountPoints := [][]string{{filepath.ToSlash(os.Getenv("HOME")), "~"}}
+	// Load mount points
+	out, err := exec.Command("mount").Output()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
This function should return an error instead of killing the program.

Reply:
Agreed as this would break future advancements in windows client support.